iot_config_client/src/settings.rs

```diff
@@ -0,0 +1,58 @@
+use helium_proto::services::{iot_config, Channel, Endpoint};
+use serde::Deserialize;
+use std::time::Duration;
+
+#[derive(Debug, Deserialize, Clone)]
+pub struct Settings {
+    /// signing key used to sign requests to iot config service
+    pub keypair: String,
+    /// grpc to iot config service
+    #[serde(with = "http_serde::uri", default = "default_url")]
+    pub url: http::Uri,
+    /// Connect timeout for follower in seconds. Default 5
+    #[serde(default = "default_connect_timeout")]
+    pub connect: u64,
+    /// RPC timeout for follower in seconds. Default 5
+    #[serde(default = "default_rpc_timeout")]
+    pub rpc: u64,
+    /// batch size for gateway stream results. Default 100
+    #[serde(default = "default_batch_size")]
+    pub batch: u32,
+}
+
+pub fn default_url() -> http::Uri {
+    http::Uri::from_static("http://127.0.0.1:8080")
+}
+
+pub fn default_connect_timeout() -> u64 {
+    5
+}
+
+pub fn default_rpc_timeout() -> u64 {
+    5
+}
+
+pub fn default_batch_size() -> u32 {
+    100
+}
+
+impl Settings {
+    pub fn connect_iot_gateway_config(&self) -> iot_config::GatewayClient<Channel> {
+        let channel = Endpoint::from(self.url.clone())
+            .connect_timeout(Duration::from_secs(self.connect))
+            .timeout(Duration::from_secs(self.rpc))
+            .connect_lazy();
+        iot_config::GatewayClient::new(channel)
+    }
+    pub fn connect_iot_region_config(&self) -> iot_config::admin_client::AdminClient<Channel> {
+        let channel = Endpoint::from(self.url.clone())
+            .connect_timeout(Duration::from_secs(self.connect))
+            .timeout(Duration::from_secs(self.rpc))
+            .connect_lazy();
+        iot_config::admin_client::AdminClient::new(channel)
+    }
```

Comment:
```suggestion
    pub fn connect_config_clients(&self) -> (iot_config::GatewayClient<Channel>, iot_config::AdminClient<Channel>) {
        let channel = Endpoint::from(self.url.clone())
            .connect_timeout(Duration::from_secs(self.connect))
            .timeout(Duration::from_secs(self.rpc))
            .connect_lazy();
        (iot_config::GatewayClient::new(channel.clone()), iot_config::AdminClient::new(channel))
    }
```
since we're connecting to both anyway, would something like this work?